lib/dust.js

```diff
@@ -115,9 +115,14 @@
     return dust.compileFn(source)(context, callback);
   };
 
+  /**
+   * Compile a template to an invokable function.
+   * If `name` is provided, also registers the template under `name`.
+   * @param source {String} template source
+   * @param name {String} template name
```

Comment:
template name is optional. i think there's a jsdoc syntax for that  `@param [name] {string}`